index.d.ts

```diff
@@ -1,17 +1,11 @@
 // * Typings for owo.js
 // *
-// * Contributed by Capuccino
+// * Contributed by Capuccino & noud02
 
-export module owo.js {
-    export interface Client {
-        key: 'key'
+export module "owo.js" {
+    export class Client {
+        public constructor (key: string);
+        public upload (path: string): Promise<any>;
```

Comment:
any could be replaced by a type alias here (object keys are in whats-this/api)

Reply:
will do, didnt have time to figure out the response so I used `any` there for now

Reply:
https://documenter.getpostman.com/view/200155/owo-api/2QsY9k